i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-core.md

```diff
@@ -16,6 +16,37 @@ notre [Github](https://github.com/centreon/centreon/issues/new/choose).
 
 ## Centreon Web
 
+### 22.04.6
+
```

Comment:
[Administration] Access group fields are not mandatory for OpenId Connect configuration